tests/test_nam/test_data.py

```diff
@@ -233,5 +235,47 @@ def _t_apply_delay_int(self, n: int, delay: int):
         return x_out, y_out
 
 
+class TestWav(object):
+    tolerance = 1e-6
+
+    @pytest.fixture(scope="class")
+    def tmpdir(self):
+        with TemporaryDirectory() as tmp:
+            yield tmp
+
+    def test_np_to_wav_to_np(self, tmpdir):
+        # Create random numpy array
+        x = np.random.rand(1000)
+        # Save numpy array as WAV file
+        filename = os.path.join(tmpdir, "test.wav")
+        data.np_to_wav(x, filename)
+        # Load WAV file
+        y = data.wav_to_np(filename)
+        # Check if the two arrays are equal
```

Comment:
Note: these are only aprpoximately equal because the default WAV format (24-bit) loses some precision on the `np` array. But it's fine.